cmd/swarmctl/service/flagparser/plugin.go

```diff
@@ -0,0 +1,18 @@
+package flagparser
+
+import (
+	"github.com/docker/swarmkit/api"
+	"github.com/spf13/pflag"
+)
+
+func parsePlugin(flags *pflag.FlagSet, spec *api.ServiceSpec) error {
```

Comment:
Okay with me, though I prefer having more specific arguments in cases like this.